cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -63,6 +63,7 @@
     {% set date_desc = controls.post_date_description or post_date_description or 'Published' %}
     {% set cat_controls = controls.categories %}
     {% set show_categories = cat_controls.show_preview_categories if cat_controls is defined else true %}
+    {% set display_settings = get_fl_display_settings(controls.categories.page_type) if cat_controls is defined else get_fl_display_settings('None') %}
```

Comment:
Since you reference `cat_controls` already in here, should this be:

`{% set display_settings = get_fl_display_settings(cat_controls.page_type) if cat_controls is defined else get_fl_display_settings('None') %}`

Reply:
Actually, this could be shortened to this, right?

```
{% set display_settings = get_fl_display_settings(cat_controls.page_type if cat_controls is defined else 'None') %}
```

Reply:
Oh, whoops. I meant to divorce this from `cat_controls` entire. I had originally piggy-backed on it, but then I realized we might be able to iterate into eliminating that `cat_controls` line and integrating that `show_preview_categories` into the `display_settings` thing I'm doing now.

I'll remove the reference to `cat_controls` in my edits.